style.css

```diff
@@ -71,12 +76,84 @@ li {
   color: #344563;
 }
 
+.popupBtnn a {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  color: #6070ff;
+}
+
+.popupBtnn a:hover {
+  color: white;
+  background-color: #6070ff;
+  border: 1px solid #6070ff;
+}
+
 .wrapper .options li a {
   color: #fff;
   font-weight: 600;
   font-size: 32px;
 }
 
+#popupCard {
+  z-index: 10;
+}
+
+.popupContaner {
+  width: 100%;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(5px);
+  background-color: rgba(185, 185, 185, 0.609);
+  z-index: 2;
+}
+
+.icons_cont li a i {
+  color: #505f79;
+}
+
+.popupContaner .popupwrapper {
+  width: 80%;
+  height: 80vh;
+  background-color: #fff;
+  border-radius: 10px;
+  overflow-y: scroll;
+}
+
+.hero-icons li a i {
+  color: #505f79;
+}
+
+.project {
+  width: 87%;
+  background-color: white;
+  border-radius: 30px;
+  margin: 57px auto;
+  padding: 16px;
+}
+
+.popupContaner .popupwrapper .project {
+  width: 100%;
+  margin: 0;
+  flex-direction: column;
+}
+
+.popupContaner .popupwrapper .project i {
+  align-self: end;
+}
```

Comment:
- [ ] OPTIONAL: I recommend you to delete this, because it is making the icons on buttons to be aligned on the bottom: 

Without `align-self: end;`  | With `align-self: end;`
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/152572007-3abdf57f-c937-4c43-b730-bf92b57884da.png)  | ![image](https://user-images.githubusercontent.com/77038610/152572037-a745fc14-6e30-47ba-9dd6-174c1d9adfb7.png)